app/assets/main/react_components/Button.jsx

```diff
@@ -6,7 +6,7 @@ export default class Button extends Component {
     const { status, className, children, onClick, href } = this.props;
 
     if (href) {
-      return(
+      return (
```

Comment:
Avoid adding extra space, check git Working Tree to see these changes and go through and revert accidental/unnecessary spaces and indents.